prepare_cyrus.hs

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env stack
+-- stack runghc --resolver lts-8.14 --package turtle --install-ghc
```

Comment:
This script is funny. I find quite good that you try to automate such task... But don't you think that using a simple bash script to do this would have been a better solution? I know less funny, but that's a rare case where a bash script could be less long that an Haskell one :)